src/main/java/com/school/project/service/impl/UserServiceImpl.java

```diff
@@ -54,4 +61,11 @@ public User getUserById(Long id) {
         if (!byId.isPresent()) throw new UserNotFoundException("User with id :" + id + "is not found");
```

Comment:
why do you need to throw twice the same error? are you getting there by any conditions?